src/rewritable_units/element.rs

```diff
@@ -53,15 +53,15 @@ pub struct Element<'r, 't> {
 }
 
 impl<'r, 't> Element<'r, 't> {
-    pub(crate) fn new(start_tag: &'r mut StartTag<'t>, can_have_content: bool) -> Self {
+    pub(crate) fn new(start_tag: &'r mut StartTag<'t>, el_can_have_content: bool) -> Self {
```

Comment:
You can also remove the `el_` here.

Reply:
Good catch, LSP rename missed that one, as did I.